tajo-core/src/main/java/org/apache/tajo/engine/planner/physical/DistinctGroupbySortAggregationExec.java

```diff
@@ -172,6 +168,13 @@ public void close() throws IOException {
 
   @Override
   public void init() throws IOException {
+    super.init();
+
+    if (aggregateExecs != null) {
```

Comment:
PhysicalPlannerImpl.createSortAggregationDistinctGroupbyExec always set the array, so it can not occurs NPE. 
Next time, I will refactor these code.

Thanks.